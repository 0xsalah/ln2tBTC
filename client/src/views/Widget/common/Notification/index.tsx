import React from 'react'

const Notification: React.FC<{ className?: string }> = (props) => {
    const { children, className = '' } = props
    return (
        <div className={`notification ${className}`}>
            <div className="notification__text">
                {children}
            </div>
        </div>
    )
}

export default Notification