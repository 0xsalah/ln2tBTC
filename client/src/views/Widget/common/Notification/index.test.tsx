import React from "react";
import { render } from "@testing-library/react";
import Notification from ".";

test("snapshot of component rendered", () => {
  const component = render(
    <Notification>
      <span>Notify!</span>
    </Notification>
  );
  expect(component.baseElement).toMatchSnapshot();
});
