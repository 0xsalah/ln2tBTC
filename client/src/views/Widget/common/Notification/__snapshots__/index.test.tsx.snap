// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of component rendered 1`] = `
<body>
  <div>
    <div
      class="notification "
    >
      <div
        class="notification__text"
      >
        <span>
          Notify!
        </span>
      </div>
    </div>
  </div>
</body>
`;
